Bugfix xdk workbench v 3 5 0

PriyaGovind opened the pull request.

wegendt-bosch commented:
jenkins test this please

wegendt-bosch commented on platforms/org.eclipse.mita.platform.xdk110/src/org/eclipse/mita/platform/xdk110/sensors/Bme280Generator.xtend:
```diff
@@ -117,7 +117,7 @@ class Bme280Generator extends AbstractSystemResourceGenerator {
             case 'temperature': codeFragmentProvider.create('''«dataVariable».temperature''')
             case 'pressure': codeFragmentProvider.create('''«dataVariable».pressure''')
-            case 'humidity': codeFragmentProvider.create('''((float)«dataVariable».humidity / 1024.0f)''')
+            case 'humidity': codeFragmentProvider.create('''((float)«dataVariable».humidity)''')
```
Since `humidity` is already in %rH and an `uint32` we should change the modality to `uint32` and remove the `fixed_point` modality. In that case please also update the documentation in `/website/site/content/platforms/xdk110.md`.

wegendt-bosch replied:
Another possibility is to change the generated C code so it exposes some kind of sub-percentage information.

PriyaGovind replied:
updated with only humidity in  %rH

wegendt-bosch commented:
Thanks. Please also update the documentation in `/website/site/content/platforms/xdk110.md` to reflect the removed modality and changed type.

PriyaGovind commented:
> Thanks. Please also update the documentation in `/website/site/content/platforms/xdk110.md` to reflect the removed modality and changed type.

Updated in xdk110.md

wegendt-bosch commented on website/site/content/platforms/xdk110.md:
```diff
@@ -26,7 +26,7 @@ Currently implemented sensors, connectivities and buses in the XDK110 platform:
 -------------------------------------------------------------------------|-------------------------------|------
 [Accelerometer]({{< ref "#accelerometer-bma280" >}})                     | [LED]({{< ref "#led" >}})     | [GPIO]({{< ref "#GPIO" >}})
-[Gyroscope]({{< ref "#gyroscope-calibrated-sensor-fusion" >}})           | [BLE]({{< ref "#ble" >}})     | [I2C]({{< ref "#I2C" >}})
+[Gyroscope]({{< ref "#gyroscope-calibrated-sensor-fusion" >}})           | [HumBLE]({{< ref "#ble" >}})     | [I2C]({{< ref "#I2C" >}})
```
Revert this please

PriyaGovind replied:
Reverted wrong change

wegendt-bosch commented on website/site/content/platforms/xdk110.md:
```diff
@@ -205,8 +205,7 @@ Name                            | Description
 `pressure : uint32`             | The pressure reported by the BME280.
-`humidity : float`              | The humidity reported by the BME280.
-`humidity_fixed_point : uint32` | The humidity reported by the BME280 in fixed-point representation: divide by 1024 to get the percentage.
+`humidity : float`              | The humidity reported by the BME280 in percentage.
```
It should read `humidity : uint32`

PriyaGovind replied:
updated
